index.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html> 
+
+    <body>
```

Comment:
Why is your HTML file not having a `head` element?
Create one with the necessary elements.
You should have a title element and link your CSS files here.